js/main.js

```diff
@@ -21,5 +21,83 @@ function submitForm(event) {
   data.entries.unshift(formObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $submit.reset();
+  document.getElementById('ul').prepend(renderEntry(formObject));
+  viewSwap('entries');
+  toggleNoEntries();
 }
 $submit.addEventListener('submit', submitForm);
+
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+
+  const $divRow = document.createElement('div');
+  $divRow.setAttribute('class', 'row');
+  $li.appendChild($divRow);
+
+  const $divCol = document.createElement('div');
+  $divCol.setAttribute('class', 'column-full column-half');
+  $divRow.appendChild($divCol);
+
+  const $imgView = document.createElement('img');
+  $imgView.setAttribute('class', 'img-view-entry');
+  $imgView.setAttribute('src', entry.photoURL);
+  $divCol.appendChild($imgView);
+
+  const $divColLower = document.createElement('div');
+  $divColLower.setAttribute('class', 'column-full column-half');
+  $divRow.appendChild($divColLower);
+
+  const $h2 = document.createElement('h2');
+  $h2.textContent = entry.title;
+  $divColLower.appendChild($h2);
+
+  const $p = document.createElement('p');
+  $p.textContent = entry.notes;
+  $divColLower.appendChild($p);
+
+  return $li;
+}
+
+function appendEntry() {
+  for (let i = 0; i < data.entries.length; i++) {
+    document.getElementById('ul').appendChild(renderEntry(data.entries[i]));
+  }
+  viewSwap(data.view);
+  toggleNoEntries();
+}
+document.addEventListener('DOMContentLoaded', appendEntry);
+
+const $noEntries = document.querySelector('.no-entries');
+function toggleNoEntries() {
+  if (data.entries.length === 0) {
+    $noEntries.setAttribute('class', 'column-full no-entries');
+  } else {
+    $noEntries.setAttribute('class', 'hidden');
+  }
+}
+
+const $entryForm = document.querySelector('.entry-form');
+const $entries = document.querySelector('.entries');
+function viewSwap(string) {
+  if (string === 'entry-form') {
+    $entryForm.setAttribute('class', 'entry-form');
+    $entries.setAttribute('class', 'hidden');
+    data.view = 'entry-form';
+  } else {
+    $entryForm.setAttribute('class', 'hidden');
+    $entries.setAttribute('class', 'entries');
+    data.view = 'entries';
+  }
+}
```

Comment:
Rather than needing to repeat `data.view = ` twice, you can just do it once outside of any conditionals and set it equal to `string`